src/dev/code_coverage/shell_scripts/fix_html_reports_parallel.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+EXTRACT_START_DIR=tmp/extracted_coverage
```

Comment:
Should this be `/tmp`?

Reply:
No

Reply:
Then should this be `tmp/`?

https://github.com/elastic/kibana/pull/62336/files/59b8cc15117017ba12a2a3042c60c154958a0e6d#diff-0910c00b7e6ad6271b6dd077d3eace3eR4